app/mixins/head-tags-static.js

```diff
@@ -36,7 +36,8 @@ export default Mixin.create({
 			gaUrl: config.tracking.ua.scriptUrl,
 			noExternals,
 			facebookAppId: config.facebook.appId,
-			lazyCss: !this.get('fastboot.isFastBoot')
+			lazyCss: !this.get('fastboot.isFastBoot'),
+			twitterSite: model.wikiVariables.twitterAccount || '@getfandom'
```

Comment:
Added in https://github.com/Wikia/app/pull/15896, `@getfandom` is the current value set in https://github.com/Wikia/app/blob/dev/includes/wikia/VariablesBase.php#L7924